doc/sphinx/proof-engine/tactics.rst

```diff
@@ -3975,15 +3973,15 @@ automatically created.
       the following. Beware, there is no operator precedence during parsing, one can
       check with :cmd:`Print HintDb` to verify the current cut expression:
 
-      .. productionlist:: regexp
-          regexp : `ident`          (hint or instance identifier)
-                 : _             (any hint)
-                 : `regexp` | `regexp` (disjunction)
-                 : `regexp` `regexp`   (sequence)
-                 : `regexp` *       (Kleene star)
-                 : emp           (empty)
-                 : eps           (epsilon)
-                 : ( `regexp` )
+      .. prodn::
+         regexp ::= @ident   (hint or instance identifier)
```

Comment:
@jfehrle Note that I needed to use non-breaking spaces in the annotation on this line to make sure that it would appear on a single line.

Reply:
Ah, nonbreaking spaces in the source file??  That's OK for a quick patch but not for the long term because it's totally invisible.  Copying the text from this screen, they appear to be normal spaces.  That could be a GitHub artifact.

Reply:
Indeed, I was not comfortable with this hack and that's why I wanted to let you know about it.